src/test/java/org/synyx/urlaubsverwaltung/absence/AbsenceApiControllerIT.java

```diff
@@ -814,26 +815,29 @@ void ensureBadRequestForMissingFromParameter() throws Exception {
     void ensureBadRequestForMissingToParameter() throws Exception {
 
         perform(get("/api/persons/23/absences")
+            .with(oidcLogin().idToken(builder -> builder.subject("muster")).authorities(new SimpleGrantedAuthority("USER")))
             .param("from", "2016-01-31"))
             .andExpect(status().isBadRequest());
     }
 
     @Test
     void ensureBadRequestForMissingPersonParameter() throws Exception {
         perform(get("/api/persons//absences")
+            .with(oidcLogin().idToken(builder -> builder.subject("muster")).authorities(new SimpleGrantedAuthority("USER")))
             .param("from", "2016-01-01")
             .param("to", "2016-01-31")
-        ).andExpect(status().isNotFound());
+        ).andExpect(status().isBadRequest());
```

Comment:
Fixed test (introduced in 908603ec )
Bad request is the actual behavior in this case and better matches the expected behavior for forgetting a parameter as implied by the method name